modules/op-resolver-git.adoc

```diff
@@ -0,0 +1,214 @@
+// This module is included in the following assembly:
+//
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_content-type: PROCEDURE
+[id="resolver-git_{context}"]
+= Specifying a remote pipeline or task from a Git repository
+
+Using the Git resolver, you can specify a remote pipeline or task from a Git repostory. The repository must contain a YAML file that defines the pipeine or task.
+
+The Git resolver can access a repository either by cloning it anonymously or else by using the authenticated SCM API.
+
+For the authenticated API, you can use Git repository providers that are supported by the link:https://github.com/jenkins-x/go-scm/tree/main/scm/driver[`go-scm` library].
+Not all `go-scm` implementations have been tested with the Git resolver, but the following providers are known to work:
+
+* github.com and GitHub Enterprise
+* gitlab.com and self-hosted Gitlab
+* Gitea
+* BitBucket Server
+* BitBucket Cloud
+
+[NOTE]
+====
+You can configure only one Git connection using the authenticated SCM API for your cluster. This connection becomes available to all users of the cluster.
```

Comment:
i would have add that you then share the token access to all users and may represent a security fix!

but i let the decision to someone else if we really need to make it scary like that cc @vdemeester